Updates README

lukecampbell opened the pull request:
This commit updates the README to include the appropriate badges and a quick developer guide for setting up the environment in conda.

lukecampbell committed:
Updates README

This commit updates the README to include the appropriate badges and a
quick developer guide for setting up the environment in conda.

kthyng commented on README.md:
```diff
@@ -1,6 +1,15 @@
+[![Code Coverage](https://img.shields.io/codecov/c/github/axiom-data-science/extract_model.svg?style=for-the-badge)](https://codecov.io/gh/axiom-data-science/extract_model)
+[![License:BSD](https://img.shields.io/badge/License-BSD-green.svg?style=for-the-badge)](https://opensource.org/licenses/MIT)
+[![Code Style Status](https://img.shields.io/github/workflow/status/axiom-data-science/extract_model/linting%20with%20pre-commit?label=Code%20Style&style=for-the-badge)](https://github.com/axiom-data-science/extract_model/actions)
+
```
you need to update the urls in the badges to this project

lukecampbell replied:
:+1: 
Good find!

lukecampbell committed:
Point at intake-erddap not extract_model

lukecampbell committed:
Update badge for License to be correct BSD version

This commit updates the LICENSE text to correctly match the text from
the BSD-2 Clause, and updates the badge to reflect that this is the
BSD-2 Clause, not the others. In the future we're looking to switch the
license to MIT, but that's for another day.
